tests/test_data/core/core__observation.txt

```diff
@@ -1,20 +1,20 @@
-([{'id': None, 'coding': [{'id': None, 'code': 'laboratory', 'display': 'Laboratory', 'system': 'http://terminology.hl7.org/CodeSystem/observation-category', 'userSelected': None, 'version': None}], 'text': None}], None, 'final', {'id': None, 'code': '34533-0', 'display': 'Odor of Urine', 'system': 'http://loinc.org', 'userSelected': None, 'version': None}, None, None, None, {'id': None, 'coding': [{'id': None, 'code': '167248002', 'display': 'Urine smell ammoniacal (finding)', 'system': 'http://snomed.info/sct', 'userSelected': None, 'version': None}], 'text': 'Urine smell ammoniacal (finding)'}, datetime.date(2018, 6, 2), datetime.date(2018, 5, 28), datetime.date(2018, 6, 1), datetime.date(2018, 1, 1), 'Patient/8877ef1f-7cd7-3242-d7f0-73cf3f7165f4', 'Encounter/299b6495-3fe7-8db3-c494-6e1ce8b7986d', '05e6184c-f3ef-017b-ce21-599d375020b3', 'Observation/05e6184c-f3ef-017b-ce21-599d375020b3')
```

Comment:
Since I did a lot of unnesting on this one, i also took this opporitunity to move the ID to the first column of the table. But there's enough going on here that a detailed comparison is probably not warranted